documentation/CAMARA-Security-Interoperability.md

```diff
@@ -279,6 +290,174 @@ Camara recommends that implementations run the OIDF interoperability suite and a
+
+In other cases, as defined in [OIDC Authentication Error Response Section](https://openid.net/specs/openid-connect-core-1_0.html#AuthError), the authorization server redirects the user-agent to the provided client redirection URI using the HTTP status code `302-Found` and includes the following `error` code parameter within the response:
+
+
+|         Error Code          | Scenario                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|:---------------------------:|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|      `invalid_request`      | The request is missing a required parameter, includes an invalid parameter value, includes a parameter more than once, or is otherwise malformed.<br/>If the server requires [Proof Key for Code Exchange (PKCE)](https://www.rfc-editor.org/rfc/rfc7636#section-4.4.1) by OAuth public clients and the client does not send the `code_challenge` in the request.<br/>If the server supporting PKCE does not support the requested `code_challenge_method`. |
+|    `unauthorized_client`    | The client is not authorized to request an authorization code using this method.                                                                                                                                                                                                                                                                                                                                                                            |
+|       `access_denied`       | The user or authorization server denied the request (for example, the user cannot be authenticated or denied the consent).                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
If the IP address cannot be mapped to a phone number, is `access_denied` the correct error code to be returned?

The example scenario: 
If the source IP address is routed to the wrong home operator and the home operator is unable to find the corresponding mobile number, to perform network authentication. Should the error code be `access_denied` or another error code?

Reply:
Auth Code flow -> access_denied
CIBA -> unknown_user_id